src/assets/cta.scss

```diff
@@ -6,14 +6,19 @@ body {
 
 button {
   &.ra21-button {
-    background-color: var(--primary-blue);
+    background-color: var(--pastel-blue);
     color: var(--white);
     border-radius: 5px;
     width: 100%;
     padding: 0;
     border: none;
   }
 
+  &.ra21-button:focus {
+    outline: none;
+    box-shadow: inset 0 0 0 1pt var(--primary-blue);
```

Comment:
Since the outline is being cut off by the IFrame, and outlines cannot have round corners without hacks, box-shadow that is inset is the best solution, since it stays contained within an element.